### PR TITLE
Update rstudio version

### PR DESCRIPTION
**Pull request by sellth**

### CODE REVIEW

File: template/script.sh.erb [hunk 1]
```diff
@@ -86,16 +86,33 @@ else
...
-  --auth-stay-signed-in-days 30 \
-  --auth-pam-helper-path "${RSTUDIO_AUTH}" \
-  --auth-encrypt-password 0 \
-  $(if [[ ! -z "${RSTUDIO_WHICH_R}" ]]; then echo --rsession-which-r "${RSTUDIO_WHICH_R}"; fi) \
-  --rsession-path "${RSESSION_WRAPPER_FILE}" \
-  --server-data-dir='/tmp/run' \
-  --server-user=$USER
- 
+if [[ "$RSTUDIO_SERVER_IMAGE" = *singularity-rstudio* ]]; then
+  singularity run --no-home -B "$BINDS" \
```
Comment (messersc) [inline template/script.sh.erb:91]: shouldn't this also be apptainer then?

**Review by messersc — CHANGES_REQUESTED** [review]
Can you change this one thing and then we are good to merge

File: template/script.sh.erb (hunk 1 above)
Comment (sellth) [reply template/script.sh.erb:91]: Left like this to make the two calls more distinguishable. This block will be remove anyway after "beta" is over.

**Comment by sellth** [comment]
This was actually intentional, but we can change it as well.

**Pull request merged**